tree/30_generic_methods/permissions_set_posix_ace.cf

```diff
@@ -0,0 +1,192 @@
+#
+#####################################################################################
+
+# @name Permissions set POSIX ace
+# @description Verify that an ace is present on a file or directory.
+# This method will append the given aces to the current POSIX ACLs of
+# the target.
+# @documentation The `permissions_*acl|ace_*` manage the POSIX ACL on files and directorieS.
+# #### Parameters
+#
+# ##### Path
+#
+# Path can be a regex with the following format:
+#
+# * * matches any filename or directory at one level, e.g. *.cf will match all files in one directory that end in .cf but it won't search across directories. */*.cf on the other hand will look two levels deep.
+# * ? matches a single letter
+# * [a-z] matches any letter from a to z
+# * {x,y,anything} will match x or y or anything.
+#
+# ##### Recursive
+# 
+# Can be:
+# 
+# * `true` to apply the given aces to folder and sub-folders and files.
+# * or `false` to apply to the strict match of `Path`
+#
+# If left blank, recursivity will automatically be set to `false`
+#
+# ##### User and Group
+#
+# ACE for user and group  can be left blank if they do not need any specification.
+# If fulfill, they must respect the format:
+#
+# `<username|groupname>:<operator><mode>`
+# 
+# with:
+#
+# * `username` being the Linux account name
+# * `groupname` the Linux group name
+# * Current `owner user` and `owner group` can be designed by the character `*`
+#
+# The operator can be:
+# * `+` to add the given ACE to the current ones.
+# * `-` to remove the given ACE to the current ones.
+# * `=` to force the given ACE to the current ones.
+#
+# You can define multiple ACEs by separating them with commas.
+# Ex: `*:=rwx, myUser:+r`
+# Will set the user ACE to `rwx`, add an exception for `myUser` if not already existing and grant him the read right.
+#
+# ##### Other
+#
+# ACE for other must respect the classic:
+# 
+# * [+-=]r?w?x?
+# It can also be left blank to let the `Other` ACE unchanged.
+#
+# @parameter path                  Path of the file or directory
+# @parameter recursive             Recursive Should ACLs cleanup be recursive, "true" or "false" (defaults to "false")
+# @parameter user                  User acls, comma separeted, like: bob:+rwx, alice:-w
+# @parameter group                 Group acls, comma separeted, like: wheel:+wx, anon:-rwx
+# @parameter other                 Other acls, like -x
+# 
+# @parameter_constraint recursive  "allow_empty_string" : true
+# @parameter_constraint user       "allow_empty_string" : true
+# @parameter_constraint group      "allow_empty_string" : true
+# @parameter_constraint other      "allow_empty_string" : true
+#
+# @parameter_constraint user       "regex" : "^$|^(([A-z0-9._-]+|\\*):[+-=]r?w?x?,? *)+$"
+# @parameter_constraint group      "regex" : "^$|^(([A-z0-9._-]+|\\*):[+-=]r?w?x?,? *)+$"
+# @parameter_constraint other      "regex" : "^$|^[+-=^]r?w?x?$"
+#
+# @class_prefix permissions_add_acl
```

Comment:
We need to chose between set and add (name vs. class/bundle name).

Will there be another method to overwrite ACLs?